lib/cbra/change.rb

```diff
@@ -74,7 +74,7 @@ def transitively_affected_components
     end
 
     def tests_to_run
-      puts "<<< Test scripts to run >>>"
+      puts "<<< Test scripts to run >>>" if selected_full_results?
```

Comment:
In hindsight, it would be nice if the tests-only output mimics cobratest. This will make it an easier drop in replacement. I also imagine others wouldn't want a title in this scenario.